.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -35,6 +35,30 @@ body:
       placeholder: ex. 7.6.0
     validations:
       required: true
+  - type: input
+    id: node-version
+    attributes:
+      label: Node version
+      description: What version of node.js are you using to run Cypress?
+      placeholder: ex. v16.16.0
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: Operating System
+      description: What operating system is Cypress running on when you are encountering your problem?  The more specific the better  ie macOS 12.4 or Windows 10.0.19044.1889
```

Comment:
Just a heads up that it looks like GitHub will not render double white space between sentences. I noticed it as an inconsistency in the doc here, but GitHub ignores it, so 🤷🏼 

Also, more minor stuff here, but we have a mix of using ending punctuation for these descriptions across the doc. I may be the only one that notices stuff like that (b/c my brain apparently likes to notice things like that), but in case you care and want to fix it, I thought I'd just mention it 😄